src/OrchardCore.Modules/OrchardCore.Forms/Drivers/ValidationPartDisplay.cs

```diff
@@ -11,6 +11,8 @@ public class ValidationPartDisplay : ContentPartDisplayDriver<ValidationPart>
     {
         public override IDisplayResult Display(ValidationPart part)
         {
+            // Empty Prefix on Display shapes
+            Prefix = "";
```

Comment:
What happens if the field that has the error is contained inside another widget, like the container widget? 
Can that happen? Does it work ok then?
Or equally if the `ValidationPart` is contained in a container does that then cause a problem because it's prefix would have included the container prefix?

Or am I finding problems where there are none?

Reply:
@deanmarcussen I guess it could be only an issue if  same`ValidationPartDisplay`  driver is used for `Display `and `Edit` in same request.

Reply:
Yes, that was my concern. I think actually it is ok, because `BuildPrefix` is called multiple times each request.
I need to look a bit further

Reply:
Ok - No change i driver - Removed from View